src/include/common/container/concurrent_bitmap.h

```diff
@@ -204,12 +206,10 @@ class RawConcurrentBitmap {
     T bits = reinterpret_cast<const std::atomic<T> *>(&bits_[*byte_pos])->load();
     if (bits == static_cast<T>(-1)) {
       *byte_pos += static_cast<uint32_t>(sizeof(T));
-      // prevent underflow
-      if (*bits_left < sizeof(T) * BYTE_SIZE) {
-        *bits_left = 0;
-      } else {
-        *bits_left = *bits_left - static_cast<uint32_t>(sizeof(T) * BYTE_SIZE);
-      }
+      // IsAlignedAndFits<T> is always called before FindUnsetBit.
```

Comment:
If you think this assumption should always be true, the right place to document this is as precondition for the method. Basically, assert that this is true at the beginning of the method.

Reply:
Like an assert statement at the beginning of the function? Or just comments?

I can also put the check back in, in case someone might call the function without checking bits_left.

Reply:
Asserts at the beginning of the function is preferred as we discussed